app/scripts/views/force_auth.js

```diff
@@ -130,9 +130,14 @@ define(function (require, exports, module) {
 
     context: function () {
       return {
+<<<<<<< HEAD
+        email: email,
+        fatalError: getFatalErrorMessage(this, fatalError),
+=======
         email: this.relier.get('email'),
         fatalError: getFatalErrorMessage(this, this._fatalError),
         isPasswordAutoCompleteDisabled: this.isPasswordAutoCompleteDisabled(),
+>>>>>>> c98f30d67969f07e229f28a1eb73af019e6be41a
         password: this._formPrefill.get('password')
```

Comment:
^ Above needs to be fixed, you need to resolve the conflict here. Basically change this file to have:
```
    context: function () {
      return {
        email: this.relier.get('email'),
        fatalError: getFatalErrorMessage(this, this._fatalError),
        password: this._formPrefill.get('password')
      };
    },
```